### ISSUE TITLE
Tensorflow is required with the latest keras core

### ISSUE THREAD

**Issue by ASEM000** [keras-team/keras-core#786]
Hello keras team,

I encountered a test failure [here](https://github.com/ASEM000/Serket/actions/runs/5957788498/job/16161045670), the issue seems to be related to importing batch norm without tensorflow installed when using using _jax_ backend,

I solved this problem by [pinning](https://github.com/ASEM000/Serket/commit/bfabec7c8c18277ecffcd270d9f24698d0f53d61) to 0.1.1

**Comment by fchollet** [comment]
Indeed, seems to have slipped by. Looking into it

**Comment by fchollet** [comment]
This should be fixed at HEAD. Note that it's still necessary to have TF in order to *build* the package, but once built you can install and run it without TF. Typically, you don't need to build the package yourself, you can just get it from PyPI (we do weekly releases so this fix will be on PyPI in 1 week).

**Comment by ASEM000** [comment]
Sounds great, thanks for the quick fix
